docs/apis-tools/camunda-api-rest/intermediate-tutorial.md

```diff
@@ -0,0 +1,239 @@
+const fileContent = fs.readFileSync(bpmnFilePath);
+formData.append("resources", fileContent, {
+  filename: "calculate-sales-tax.bpmn",
+  contentType: "application/xml",
+});
+```
+
+7. Configure your POST request to the appropriate endpoint, including an authorization header based on the previously acquired `accessToken`:
+
+```javascript
+const options = {
+  method: "POST",
+  url,
+  headers: {
+    Accept: "application/json",
+    Authorization: `Bearer ${accessToken}`,
+    ...formData.getHeaders(),
+  },
+  data: formData,
+};
+```
+
+8. Call the endpoint, process the results from the API call, and emit an error message from the server if necessary:
+
+```javascript
+try {
+  const response = await axios(options);
+  const deployedResources = response.data.deployments;
+
+  // Emit deployed resources
+  deployedResources.forEach((x) =>
+    console.log(
+      `Process Definition Key: ${x.processDefinition.processDefinitionKey}; Process Definition Id: ${x.processDefinition.processDefinitionId}`
+    )
+  );
+} catch (error) {
+  // Emit an error from the server.
+  console.error(error.message);
+}
+```
+
+9. In your terminal, run `node cli.js processInstances deploy`.
+
+:::note
+This `deploy` command is connected to the `deployResources` function at the bottom of the `camunda-process-instances.js` file, and executed by the `cli.js` file. While we will work with roles in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+The existing process definition key and ID will now output. If you have an invalid API name or action name, or no arguments provided, or improper/insufficient credentials configured, an error message will output as outlined in the `cli.js` file.
+
+## Create and start a process instance (POST)
+
+To create and start a process instance based on the process instance key obtained in the request above, take the following steps:
+
+1. Outline your function, similar to the steps above:
+
+```javascript
+async function createInstance([processDefinitionKey]) {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+  const camundaApiUrl = process.env.CAMUNDA_REST_ADDRESS;
+  const url = `${camundaApiUrl}/process-instances`;
+}
+```
+
+2. Configure the API call:
+
+```javascript
+const options = {
+  method: "POST",
+  url,
+  headers: {
+    Accept: "application/json",
+    Authorization: `Bearer ${accessToken}`,
+  },
+  data: {
+    processDefinitionKey: processDefinitionKey,
+    variables: {
+      total: 90.0,
+    },
+  },
+};
+```
+
+3. Process the results from the API call. For example:
+
+```javascript
+try {
+  const response = await axios(options);
+  const processInstance = response.data;
+
+  console.log(`Process Instance Key: ${processInstance.processInstanceKey}`);
+} catch (error) {
+  console.error(error.message);
+}
+```
+
+4. In your terminal, run `node cli.js processInstances create <key>`, where `<key>` is the process definition key.
```

Comment:
non-blocking: add something about seeing the processInstanceKey in the output, and capturing it for the final method?